spec/factories.rb

```diff
@@ -245,7 +245,8 @@
 
   factory :sub_section do
     title { Faker::Lorem.sentence }
-    content { "(#{Faker::Lorem.sentence})[/#{File.join(Faker::Lorem.words)}]" }
+    content { "[#{Faker::Lorem.sentence}](/#{File.join(Faker::Lorem.words)})" }
+    position { rand(4) }
```

Comment:
Would there be a problem if two sub_sections had the same position. If not, this is fine. If it would be problematic, I wonder if something like this would be better:
```ruby
position {  (SubSection.maximum(:position) || 0) + 1 }
```